internal/statemachine/testdata/image_definitions/test_deb822.yaml

```diff
@@ -0,0 +1,85 @@
+name: ubuntu-server-amd64
+display-name: Ubuntu Server amd64
+revision: 1
+architecture: amd64
+series: jammy
+class: preinstalled
+kernel: linux-image-generic
+gadget:
+  url: "https://github.com/snapcore/pc-gadget.git"
+  branch: classic
+  type: "git"
+rootfs:
+  components:
+    - main
+    - universe
+    - restricted
+  sources-list-deb822: true
+  seed:
+    urls:
+      - "git://git.launchpad.net/~ubuntu-core-dev/ubuntu-seeds/+git/"
+      - "git://git.launchpad.net/~ubuntu-core-dev/ubuntu-seeds/+git/"
+    branch: jammy
+    names:
+      - server
+      - minimal
+      - standard
+      - cloud-image
+customization:
```

Comment:
Since this image definition is just here to test the deb822 warning, I think you can remove almost everything starting from this line. Only keeping a simplify `artifacts` section should be good.

Reply:
Okay